lib/mimic/cover.ex

```diff
@@ -11,12 +11,21 @@ defmodule Mimic.Cover do
     :cover.is_compiled(module) != false
   end
 
+  @doc false
+  def private_functions_exported? do
```

Comment:
Should this function be private?

Reply:
yeah might as well since export_private_functions is idempotent